src/displayapp/DisplayApp.cpp

```diff
@@ -316,6 +316,7 @@ void DisplayApp::LoadApp(Apps app, DisplayApp::FullRefreshDirections direction)
       break;
     case Apps::None:
     case Apps::Clock:
+    default:
```

Comment:
I think this change may hide warnings we want to have. Unhandled screens shouldn't just silently be ignored. A warning can show us where we have forgotten to change something

Reply:
Hmm, that's true, however entering an app instead going to the watchface is a pretty obvious issue as well, so it's not completely "silent". Maybe I can instead remove the unused Weather app from the Apps enum?

Reply:
I think I remember @Avamander saying that they didn't handle the Weather app in that switch on purpose. I can't remember the reasoning behind it anymore, but maybe they could shed some light?

Reply:
yes, removing the enum entry would feel "more right" for me

Reply:
I think @Avamander did not add the weather app because it's mostly a work in progress and/or a demo application, and not something we want the user to use.

Reply:
Why not add it to the switch and just not add it to the app list? It's more work to enable it otherwise. Should I add it there?

Reply:
I'd say, yes add it to the switch statement, and later when it is ready the screen will be made reachable through the interface

Reply:
Looks like WeatherService isn't accessible in DisplayApp currently as required by the Weather app. I'm not sure how this app is intended to be used or what's the best approach to resolve this issue.

Reply:
Then I'd say, don't touch the switch statement, to stop blocking the remaining fixed warnings. We can touch that at a later time

Reply:
Yeah, JF002 is correct, the Weather app is very much a debug utility at this moment. 

If someone touched up the UI, it could be enabled by default, but I wanted to get the already dragged on for long PR to have a fixed and reachable feature scope.